AdyenDropIn/DropInComponent.swift

```diff
@@ -172,21 +171,25 @@ public final class DropInComponent: NSObject,
     }
     
     private func didSelectCancelButton(isRoot: Bool, component: PresentableComponent) {
-        guard !paymentInProgress else { return }
+        guard !paymentInProgress || component is Cancellable else { return }
         
         if isRoot {
             self.delegate?.didFail(with: ComponentError.cancelled, from: self)
         } else {
             navigationController.popViewController(animated: true)
-            stopLoading(withSuccess: true, completion: nil)
```

Comment:
StopLoading has to be called here for issuer list component and payment method list component to stop spinning if user cancels